src/aleph/web/controllers/utils.py

```diff
@@ -66,6 +68,21 @@ def iter_pages(self, left_edge=2, left_current=2, right_current=5, right_edge=2)
                 last = num
 
 
+def make_date_filters(start: float, end: float, filter_key: str):
+    filters = []
+    if start:
+        filters.append({filter_key: {"$gte": start}})
+    if end:
+        filters.append({filter_key: {"$lt": end}})
+
+    if len(filters) > 1:
+        return {"$and": filters}
+    if filters:
```

Comment:
```suggestion
    if filters:
        assert len(filters) == 1
```

Reply:
Don't need to, the `if` above catches lists with multiple elements.